server/views/layout.pug

```diff
@@ -6,29 +6,37 @@ html(lang="en")
         meta(property="og:image" content="/public/images/opengraph-image.png?0.19.2")
         title Licences Application
 
-
-        <!--[if gt IE 8]><!--><link href="/public/stylesheets/govuk-template.css?"+version media="screen" rel="stylesheet" type="text/css" /><!--<![endif]-->
-
-        <!--[if gte IE 9]><!--><link href="/public/stylesheets/fonts.css?0.19.2" media="all" rel="stylesheet" /><!--<![endif]-->
-        <!--[if IE 8]><link href="/public/stylesheets/govuk-template-ie8.css?0.19.2" media="screen" rel="stylesheet" /><!--<![endif]-->
-
-        <!--[if lt IE 9]><script src="/public/javascripts/ie.js?0.19.2"></script><!--<![endif]-->
-
         link(rel="shortcut icon", href="/public/images/favicon.ico?0.19.2", type="image/x-icon")
         link(rel="mask-icon", href="/public/images/gov.uk_logotype_crown.svg?0.19.2", color="#0b0c0c")
         link(rel="apple-touch-icon-precomposed", sizes="152x152", href="/public/images/apple-touch-icon-152x152.png?0.19.2")
         link(rel="apple-touch-icon-precomposed", sizes="120x120", href="/public/images/apple-touch-icon-120x120.png?0.19.2")
         link(rel="apple-touch-icon-precomposed", sizes="76x76", href="/public/images/apple-touch-icon-76x76.png?0.19.2")
         link(rel="apple-touch-icon-precomposed", sizes="76x76", href="/public/images/apple-touch-icon-60x60.png?0.19.2")
+
+        <!--[if gt IE 8]><!--><link href="/public/stylesheets/govuk-template.css?"+version media="screen" rel="stylesheet" type="text/css" /><!--<![endif]-->
```

Comment:
Mainly re-ordering stuff so that we have template stuff first followed by our CSS